bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/model/WildCardParameter.java

```diff
@@ -1,6 +1,6 @@
 package org.bahmni.module.bahmnicommons.model;
 
-import org.apache.commons.lang.StringEscapeUtils;
+import org.bahmni.module.bahmnicommons.util.SqlQueryHelper;
 
 public class WildCardParameter {
```

Comment:
Only used in PatientNameQueryHelper, which itself is redundant as it is used from deprecated PatientDao.getPatients() method